### PR TITLE
Focused workspaces documention and blog post

### PR DESCRIPTION
**Pull request by bdwain**
cc @bestander

### CODE REVIEW

File: lang/en/docs/workspaces.md [hunk 1]
```diff
@@ -90,9 +90,11 @@ Yarn's workspaces are the low-level primitives that tools like Lerna can (and [d
   workspaces-experimental false
   ```
 
+* If you're only making changes to a single workspace, use [--focus](/blog/2018/05/17/focused-workspaces) to quickly install sibling dependencies from the registry rather than building all of them from scratch.
+
 ### Limitations & Caveats <a class="toc" id="toc-limitations-caveats" href="#toc-limitations-caveats"></a>
 
-* The package layout will be different between your workspace and what your users will get (the workspace dependencies will be hoisted higher into the filesystem hierarchy). Making assumptions about this layout was already hazardous since the hoisting process is not standardized, so theoretically nothing new here. If you encounter issues, try using [the `nohoist` option](https://yarnpkg.com/blog/2018/02/15/nohoist/)
+* The package layout will be different between your workspace and what your users will get (the workspace dependencies will be hoisted higher into the filesystem hierarchy). Making assumptions about this layout was already hazardous since the hoisting process is not standardized, so theoretically nothing new here. If you encounter issues, try using [the `nohoist` option](/blog/2018/02/15/nohoist/)
```
Comment (bdwain) [inline lang/en/docs/workspaces.md]: just made this link relative so that it worked locally too.

**Comment by bestander** [comment]
Great job!
Do you have a twitter handler for me to mention?

File: _posts/2018-05-18-focused-workspaces.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+date       : 2018-05-18 00:00:00
+categories : announcements
+share_text : "Ease the Transition to a Monorepo with Focused Workspaces"
+---
+
+In a [previous post](/blog/2017/08/02/introducing-workspaces/), we wrote about monorepos and how Yarn Workspaces makes working with them much simpler. Unfortunately, moving to a monorepo is not always an easy choice because without the right tooling, they can often harm the developer exerpience instead of help it.
+
+## The Problem With Monorepos
+
+One of the main reasons projects usually move to a monorepo is because they make it much easier to make changes across multiple packages by allowing you make those changes in a single pull request. One of the ways Yarn Workspaces helps improves this workflow is by automatically symlinking sibling package dependencies in the node_modules folder, allowing you to immediately see the results of a change to one package when working on another.
```
Comment (bestander) [inline _posts/2018-05-18-focused-workspaces.md]: Yarn Workspaces helps improves this => Yarn Workspaces improves this

**Comment by Haroenv** [comment]
Deploy preview for *yarnpkg* ready!

Built with commit d2d025a551a9b1e285b0467d4e577772687b245b

https://deploy-preview-820--yarnpkg.netlify.com

**Comment by bdwain** [comment]
Yep it's also @bdwain. I'm not that active on there so I linked my github in the post, but you can mention my twitter as well.

File: _posts/2018-05-18-focused-workspaces.md (hunk 2 above)
Comment (bdwain) [reply _posts/2018-05-18-focused-workspaces.md]: fixed

**Comment by bestander** [comment]
Do you think we should wait for a next release before publishing the blog?
Then people could try your feature straight a way

**Comment by Haroenv** [comment]
Deploy preview for *yarnpkg* ready!

Built with commit 5c4a2785fe95fce7d006618f4c08040d16d51597

https://deploy-preview-820--yarnpkg.netlify.com

**Comment by bdwain** [comment]
Yea that would make sense. I mentioned in the blog post that it would be ready in 1.7.0 (since that's the next minor release) but I guess there's not much reason to post about it before people can use it.

What do you think the timeline is on doing a release? Do you typically just wait a certain amount of time since the last one or just wait until enough new features are ready or something?

**Comment by Haroenv** [comment]
Deploy preview for *yarnpkg* ready!

Built with commit 2df3b641eb98799683abab7f98b24313643ec5ab

https://deploy-preview-820--yarnpkg.netlify.com

**Comment by bestander** [comment]
I think 1.7 is due to be launched.
Let me ping the team

**Comment by bdwain** [comment]
Sounds good!

**Comment by bestander** [comment]
1.7 is out 💯

**Comment by bdwain** [comment]
Sweet! thanks for the help!

**Comment by bdwain** [comment]
@bestander hm something may have gone wrong with the release. homebrew is still on yarn 1.6, the website has not been updated, and i saw this in the commit history

screen shot 2018-05-23 at 5 04 46 pm

**Comment by bdwain** [comment]
actually maybe things were just cached or hadn't propagated yet. Everything seems good now except homebrew is behind. I'll see if it's still behind in an hour or so

**Comment by bdwain** [comment]
all good!

**Pull request merged**